migrations/20190708151237_alter_state_in_permissions_requests.down.sql

```diff
@@ -0,0 +1,3 @@
+DROP TYPE permission_request_state CASCADE;
+ALTER TABLE permissions_requests DROP COLUMN state;
+ALTER TABLE permissions_requests ADD COLUMN state smallint NOT NULL DEFAULT 0;
```

Comment:
Do you need the default? How big is the permissions_requests table? Adding a NULLABLE column is probably much easier in terms of database impact.

Same below.